guides/introduction/welcome.md

```diff
@@ -0,0 +1,143 @@
+|-----------------|--------------------------|-----------|--------------------------------------------------------|
+| `_format`       | swiftui, jetpack, html   | ✅        | The content type to be processed by LiveView Native    |
+| `_interface`    | mobile, watch, tv        |           | The general device type                                |
+| `os_version`    | `string`                 |           | The version of the client OS                           |
+| `app_version`   | `string`                 |           | The version of the client application                  |
+
+This is formatted as `/?_format=xx&_interface=xx&os_version=xx&app_version=xx`, and when no query parameters are provided,
+will default to the corresponding Phoenix LiveView route (presuming a route is provided).
+
+### Processing
+
+Once a request is sucessfully delegated by LiveView Native, it will attempt to match on your LiveView route.
+
+By design, LiveView Native does not ship with a default rendering package, and instead seperates
+itself into a series of distinct packages. Each package ships with its own modifiers to handle its respective client,
+and unlike many framework agnostic development frameworks, intentionally seperates your markup by platform.
+
+This is to maintain instant feature parity with your platform(s) of choice, as LiveView Native is not concerned with
+cross-platform "bridging." This allows LiveView Native to serve native code to the platform target
+with no changes to the underlying client.
+
+So long as your client supports the underlying components sent back, it will render!
+
+Let's see an example.
+
+```elixir
+# This entry point to your LiveView, which can handle events from any platform
+# lib/my_app_web/live/hello_live.ex
+defmodule MyAppWeb.HelloLive do
+  use MyAppWeb, :live_view
+  use MyAppNative, :live_view
+
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+end
+
+# This module will be called on if the format is :swift_ui
+# lib/my_app_web/live/hello_live_swiftui.ex
+defmodule MyAppWeb.HelloLive.SwiftUI do
+  use MyAppNative, :live_view
+
+  # Within formats, you can target sub-platforms, offering versatility in your views
+  def render(assigns, %{"target" => "watchos"}) do
+    ~LVN"""
+    <VStack>
+      <Text>
+        Hello WatchOS!
+      </Text>
+    </VStack>
+    """
+  end
+
+  def render(assigns, _interface) do
+    ~LVN"""
+    <VStack>
+      <Text>
+        Hello SwiftUI!
+      </Text>
+    </VStack>
+    """
+  end
+end
+
+# This module will be called on if the format is :jetpack
+# lib/my_app_web/live/hello_live_jetpack.ex
+defmodule MyAppWeb.HelloLive.Jetpack do
+  use MyAppNative, :live_view
+
+  def render(assigns, _interface) do
+    ~LVN"""
+    <Text>Hello Jetpack Compose!</Text>
+    """
+  end
+end
+```
+
+The modules above show a distinct path to each format handled by its respective rendering package.
```

Comment:
it may be good to show a mermaid diagram to visualize the delegation and how baserd upon the `_format` value render delegation will be handled by the correct render component. Seeing the request with no `_format` use the regular LiveView `render/1` then with `_format=swiftui` being delegated to the correct render component and so on